Documentation/PageTranslations/Index.rst

```diff
@@ -29,9 +29,9 @@ The form with the fields for the translated page will then be displayed:
 
     Editing the page fields for an alternative page language
 
-Most of the fields are the same as for the page of the default language. Below
+All of the fields are the same as for the page of the default language. Below
 the fields the content of the default language is shown.
 
-If a page has no translation (that means, no "alternative page language" record),
+If a page has no translation,
```

Comment:
```suggestion
If a page has no translation meta-data,
```

Maybe this better reflects that it's about a "pages" database entry here, without resorting to use "record"?

Reply:
The term 'meta-data' is not understandable.

Reply:
What do you suggest? Just "translation" is wrong, because here it's about a proxy entry for the language in "pages", and not the CONTENT of it. That's why I picked "meta-data", but maybe you have another idea?